client_cpp/src/posts/models.hh

```diff
@@ -172,6 +174,8 @@ public:
         sticky = false,
          // Thread is locked. Only for OPs.
         locked = false,
+        // Red/Blue text is disabled.
+        rbText = false,
```

Comment:
Why is this on the Post model? Please put it on `state.hh::BoardConfig` and add the appropriate field to the JSON parsing constructor.